internal/controllers/ipaddressclaim.go

```diff
@@ -21,23 +21,16 @@ import (
 	"fmt"
 
 	"github.com/pkg/errors"
+	"golang.org/x/exp/slices"
```

Comment:
If we bump to go v1.21 we can use the builtin slices: https://pkg.go.dev/slices#ContainsFunc

Reply:
Updated go to 1.21